code/game/objects/structures/crates_lockers/closets/secure/guncabinet.dm

```diff
@@ -39,11 +40,9 @@
 				gun.pixel_x = i*4
 				overlays += gun
 
-		overlays += icon(src.icon,"door")
+		overlays += "door"
```

Comment:
If you could do a sweep and convert `overlays +=` to `add_overlay()` and `overlays -=` to `remove_overlay()`, that would be nice.